source/develop/api/design/vdsm-api.html.md

```diff
@@ -31,7 +31,7 @@ Key design points:
 *   No business logic will be implemented in the QMF agent or REST API
 *   Both QMF and REST are optional components (ie. they are not required for vdsm to function correctly). However at least one of these is needed to enable management.
 
-![](Vdsm-api-architecture.png‎ "Vdsm-api-architecture.png‎")
+![](Vdsm-api-architecture.png "Vdsm-api-architecture.png")
```

Comment:
often it is junk but sometimes not, so I cannot automate this without destroying content